Libraries/LibLine/Editor.cpp

```diff
@@ -445,7 +437,7 @@ String Editor::get_line(const String& prompt)
                 m_refresh_needed = true;
                 continue;
             }
-            if (ch == 0xc) {                    // ^L
```

Comment:
oh ffs...someone give me a clang-format configuration that doesn't do this braindead thing